internal/ingress/controller/store/store_test.go

```diff
@@ -605,35 +607,95 @@ func TestStore(t *testing.T) {
 	// check invalid secret (missing ca)
 }
 
-func createNamespace(clientSet *kubernetes.Clientset, t *testing.T) string {
+func createNamespace(clientSet *fake.Clientset, t *testing.T) string {
 	t.Log("creating temporal namespace")
-	ns, err := framework.CreateKubeNamespace("store-test", clientSet)
+
+	namespace := &v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "store-test",
+		},
+	}
+
+	ns, err := clientSet.CoreV1().Namespaces().Create(namespace)
 	if err != nil {
-		t.Errorf("unexpected error creating ingress client: %v", err)
+		t.Errorf("unexpected error creating the namespace: %v", err)
 	}
 	t.Logf("temporal namespace %v created", ns)
 
-	return ns
+	return ns.Name
 }
 
-func deleteNamespace(ns string, clientSet *kubernetes.Clientset, t *testing.T) {
-	t.Logf("deleting temporal namespace %v created", ns)
-	err := framework.DeleteKubeNamespace(clientSet, ns)
+func deleteNamespace(ns string, clientSet *fake.Clientset, t *testing.T) {
+	t.Logf("deleting temporal namespace %v", ns)
+
+	err := clientSet.CoreV1().Namespaces().Delete(ns, &metav1.DeleteOptions{})
 	if err != nil {
-		t.Errorf("unexpected error creating ingress client: %v", err)
+		t.Errorf("unexpected error deleting the namespace: %v", err)
 	}
 	t.Logf("temporal namespace %v deleted", ns)
 }
 
-func ensureIngress(ingress *extensions.Ingress, clientSet *kubernetes.Clientset) (*extensions.Ingress, error) {
-	s, err := clientSet.ExtensionsV1beta1().Ingresses(ingress.Namespace).Update(ingress)
+func createConfigMap(clientSet *fake.Clientset, ns string, t *testing.T) string {
+	t.Log("creating temporal config map")
```

Comment:
This is a helper function, adding `t.Helper()` here would allow logs to report line numbers accurately.